packages/frontend/src/MobileRoutes.tsx

```diff
@@ -200,6 +202,11 @@ const PUBLIC_ROUTES: RouteObject[] = [
         path: '/no-mobile-page',
         element: <MobileView />,
     },
+    {
+        // Autoclose popup after github installation
+        path: '/generalSettings/integrations',
```

Comment:
this is harcoded in the git integration, and we can't change this , so I added this to mobile routes to autoclose it